Polyfill.md

```diff
@@ -70,6 +70,10 @@ Some divergences that we've identified as potentially desirable:
   standard behavior:
   - Return zero when conversion from floating point to integer fails;
   - Optionally canonicalize NaN values.
+* **[NaN bit-pattern propagation](AstSemantics.md#floating-point-operators)**:
+  Regardless of WebAssembly behavior, an asm.js polyfill will follow its
```

Comment:
It's not really because the polyfill is asm.js that NaN behavior isn't specified. Just say "JavaScript polyfill" here instead?